src/views/app/folder-panel/messages/message-list-component.tsx

```diff
@@ -145,8 +147,18 @@ export const MessageListComponent: FC<MessageListComponentProps> = memo(
 			if (isSearchModule) {
 				return '';
 			}
-			return folder?.absFolderPath?.split('/')?.join(' / ') ?? '';
-		}, [folder?.absFolderPath, isSearchModule]);
+			return (
+				folder?.absFolderPath
+					?.split('/')
+					?.map((p, idx) =>
+						getFolderTranslatedName({
+							folderId: idx === 1 ? root?.id : folderId,
+							folderName: p
+						})
+					)
+					.join(' / ') ?? ''
```

Comment:
Hi @mbarto, this string logic is equal to the one in `conversation-list-component.tsx`. Can you extract a function?